swarm/pss/pss.go

```diff
@@ -330,11 +330,11 @@ func (p *Pss) handlePssMsg(msg interface{}) error {
 		return fmt.Errorf("invalid message type. Expected *PssMsg, got %T ", msg)
 	}
 	if int64(pssmsg.Expire) < time.Now().Unix() {
-		log.Trace(fmt.Sprintf("pss filtered expired message FROM %x TO %x", p.Overlay.BaseAddr(), common.ToHex(pssmsg.To)))
+		log.Trace(fmt.Sprintf("pss filtered expired message FROM %s TO %s", common.ToHex(p.Overlay.BaseAddr()), common.ToHex(pssmsg.To)))
```

Comment:
We have structured logs, if we are going to log strings, no need to use `fmt.Sprintf` at all, just use the fields of the logger.